src/sentry/api/helpers/group_index/delete.py

```diff
@@ -110,6 +108,31 @@ def delete_group_list(
         )
 
 
+def delete_group_hashes(
+    project_id: int,
+    group_ids: Sequence[int],
+) -> None:
+    batch_size = options.get("deletions.group-hashes-batch-size")
+
+    group_hashes = list(
+        RangeQuerySetWrapper(
+            GroupHash.objects.filter(project_id=project_id, group_id__in=group_ids),
```

Comment:
We will be querying all group IDs (`group_id__in=group_ids`) rather than one group at a time.